main/src/main/java/com/google/android/apps/muzei/util/SvgPathParser.java

```diff
@@ -149,6 +149,21 @@ public Path parsePath(String s) throws ParseException {
                     mCurrentPoint.set(tempPoint1);
                     break;
                 }
+                
```

Comment:
Could you remove the extra spaces on this line to match the code format of the rest of the file?